.ci/run_metrics_PR_ci.sh

```diff
@@ -93,15 +93,21 @@ check() {
 		else
 			# For bare metal repeatable machines, the config file name is tied
 			# to the uname of the machine.
-			local CM_BASE_FILE="${CHECKMETRICS_CONFIG_DIR}/checkmetrics-json-$(uname -n).toml"
+			local CM_BASE_FILE="${CHECKMETRICS_CONFIG_DIR}/checkmetrics-json-${KATA_HYPERVISOR}-$(uname -n).toml"
 		fi
 
+
 		checkmetrics --debug --percentage --basefile ${CM_BASE_FILE} --metricsdir ${RESULTS_DIR}
 		cm_result=$?
 		if [ ${cm_result} != 0 ]; then
 			echo "checkmetrics FAILED (${cm_result})"
 			exit ${cm_result}
 		fi
+
+                # Save results
+		sudo mkdir -p "${RESULTS_DIR}/${KATA_HYPERVISOR}"
+		echo "Move results"
+		sudo mv ${RESULTS_DIR}/*.json ${RESULTS_DIR}/${KATA_HYPERVISOR}
```

Comment:
`save results` should be part of the TRAP and since jenkins is unable to save folders, you should rename them

```
for f in ${RESULTS_DIR}/*.json; do mv -- "$f" "${RESULTS_DIR}/${KATA_HYPERVISOR}-$(basename $f)"; done
```